test/e2e/template/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: {{.Name}}
+    aadpodidbinding: {{.IdentityBinding}}
+  name: {{.Name}}
+  namespace: default
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{.Name}}
+        aadpodidbinding: {{.IdentityBinding}}
+    spec:
+      containers:
+      - name: {{.Name}}
+        image: chewong/identityvalidator:1.0
+        command:
+          - sleep
+          - "3600"
+        imagePullPolicy: Always
+        env:
+        - name: MY_POD_NAME
```

Comment:
nit: Can you please convert to more meaningful name like E2ETestName and E2EPodNamespace.